docs/zh/reference/sql/ddl/SET_STATEMENT.md

```diff
@@ -9,15 +9,16 @@ SetStatement ::=
 variableName ::=
 		|	sessionVariableName
 	
-sessionVariableName ::= '@@'Identifier | '@@session.'Identifier
+sessionVariableName ::= '@@'Identifier | '@@session.'Identifier | '@@global.'Identifier
```

Comment:
another syntax for set statement:
```
'SET' [ GLOBAL | SESSION ] <variableName> '=' <value>
```

better noted as well

Reply:
ok